contracts/exchange/interfaces/IRoyaltyManager.sol

```diff
@@ -34,7 +39,15 @@ interface IRoyaltyManager {
         uint256[] calldata _royaltyFees
     ) external;
 
+    function transferRoyalty(
```

Comment:
make the single one "transferRoyalty" and the batch function "transferRoyalties"